### ISSUE TITLE
Server Side pagination with UI

### ISSUE THREAD

**Issue by dharmeshsharma** [generic-ui/generic-ui#35]
Hello thanks for grid and I'm using but I have some huge data so i wan to load when UI page is change i tried to find in your documents but not find the way and event. can you guide or if this feature is no there plz add this also.

**Comment by mdymel** [comment]
hi, did you manage to solve this? I can't find such option in the documentation.

**Comment by kokahonta** [comment]
NO, I have changed to another way and make own pagination to handle this with other grid.
